src/main/java/com/enderio/EnderIO.java

```diff
@@ -104,10 +105,7 @@ public EnderIO() {
         new PaintingTrigger().register();
 
         // Decor
-        DecorBlocks.register();
-        DecorBlockEntities.register();
-        DecorEntities.register();
-        EnderDecorNetwork.register();
+        BaseNetwork.register();
```

Comment:
Can we move this up with the others, drop the Decor comment and maybe call BaseNetwork EIONetwork for consistency?